packages/legend-query-builder/src/stores/milestoning/QueryBuilderMilestoningState.ts

```diff
@@ -155,6 +155,9 @@ export class QueryBuilderMilestoningState implements Hashable {
     if (
       !this.queryBuilderState.parametersState.parameterStates.find(
         (p) => p.variableName === parameterName,
+      ) &&
```

Comment:
lets also look anywhere else in the code base where we look for variables to ensure we have looked inside the constant states as well.

Reply:
I don't think we have this kind of logic anywhere else. Checked it too. Milestoning is an excpetion because we magically create parameters for them to use in the query